install.sh

```diff
@@ -6,7 +6,7 @@ OPERATOR_PACKAGE=${OPERATOR_PACKAGE:-service-binding-operator}
 DOCKER_CFG=${DOCKER_CFG:-$HOME/.docker/config.json}
 CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-docker}
 
-if [ "$(kubectl get crd clusterserviceversions.operators.coreos.com &>1 || echo false)" == "false" ]; then
+if [ "$(kubectl get crd clusterserviceversions.operators.coreos.com &>/dev/null || echo false)" == "false" ]; then
```

Comment:
IMHO, it would be good to see errors, it can help with debugging. Hence, the fix could add the missing `&` in front of `1`

While we are here, this `if` block can be simplified further:

```
kubectl get crd clusterserviceversions.operators.coreos.com || \
  curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v${OLM_VERSION}/install.sh | bash -s v${OLM_VERSION}
```

Reply:
@pedjak  `&>&1` would not work (https://stackoverflow.com/questions/24793069/what-does-do-in-bash/24793436#24793436)

But I take your further suggestion. Thx